jvm-statistics/src/main/java/com/criteo/hadoop/garmadon/jvm/statistics/NetworkStatistics.java

```diff
@@ -2,86 +2,216 @@
 
 import com.criteo.hadoop.garmadon.jvm.AbstractStatistic;
 import com.criteo.hadoop.garmadon.jvm.StatisticsSink;
-import oshi.SystemInfo;
-import oshi.hardware.NetworkIF;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.RandomAccessFile;
+import java.lang.management.ManagementFactory;
 import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 class NetworkStatistics extends AbstractStatistic {
     private static final String NETWORK_HEADER = "network";
```

Comment:
I prefer to keep it in the process section. Also we are performing some device split on readers/elasticsearch/src/main/java/com/criteo/hadoop/garmadon/elasticsearch/EventHelper.javabased on disk and network section name that we don't want to apply here